2018 Robot/src/org/usfirst/frc/team4999/commands/autonomous/MoveDistance.java

```diff
@@ -0,0 +1,141 @@
+package org.usfirst.frc.team4999.commands.autonomous;
+
+import org.usfirst.frc.team4999.robot.MoPrefs;
+import org.usfirst.frc.team4999.robot.Robot;
+import org.usfirst.frc.team4999.robot.RobotMap;
+import org.usfirst.frc.team4999.robot.sensors.ADIS16448_IMU;
+import org.usfirst.frc.team4999.robot.sensors.VMXPi;
+import org.usfirst.frc.team4999.robot.subsystems.DriveSystem;
+
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.PIDController;
+import edu.wpi.first.wpilibj.PIDOutput;
+import edu.wpi.first.wpilibj.PIDSource;
+import edu.wpi.first.wpilibj.PIDSourceType;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class MoveDistance extends Command {
+	
+	private PIDController pid;
+	private Encoder left = RobotMap.leftDriveEncoder;
+	private Encoder right = RobotMap.rightDriveEncoder;
+	private double distance;
+	private static double ticksPerMeter = 1;
```

Comment:
Not even close to correct value.  Should probably be configurable, not hardcoded, although it's unlikely to change frequently.  Will drift based on tire wear.